support-frontend/assets/helpers/formValidation.js

```diff
@@ -13,6 +13,10 @@ import { Canada, UnitedStates } from './internationalisation/countryGroup';
 export const emailRegexPattern = '^[a-zA-Z0-9\\.!#$%&\'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)+$';
 
 export const isNotEmpty: (string | null) => boolean = input => !!input && input.trim() !== '';
+
+export const isEmpty: (string | null) => boolean = input =>
```

Comment:
We should be able to redefine `isNotEmpty` as `! isEmpty()` now shouldn't we?

Reply:
@rupertbates That was my first point of call, but we figured make it explicit.

Reply:
actually rupert is suggesting something different, we originally had !isNotEmpty and isNotEmpty, he is suggesting isEmpty and !isEmpty